hamocc/mo_ncout_hamocc.F90

```diff
@@ -585,6 +594,12 @@ subroutine ncwrt_bgc(iogrp)
     call wrtsrf(jcalflx2000(iogrp),  FLX_CAL2000(iogrp),  rnacc*1e3/dtbgc,0.,cmpflg,'calflx2000')
     call wrtsrf(jcalflx4000(iogrp),  FLX_CAL4000(iogrp),  rnacc*1e3/dtbgc,0.,cmpflg,'calflx4000')
     call wrtsrf(jcalflx_bot(iogrp),  FLX_CAL_BOT(iogrp),  rnacc*1e3/dtbgc,0.,cmpflg,'calflx_bot')
+    call wrtsrf(jdustflx0100(iogrp),  FLX_DUST0100(iogrp),  rnacc*1e3/dtbgc,0.,cmpflg,'dustflx0100')
+    call wrtsrf(jdustflx0500(iogrp),  FLX_DUST0500(iogrp),  rnacc*1e3/dtbgc,0.,cmpflg,'dustflx0500')
+    call wrtsrf(jdustflx1000(iogrp),  FLX_DUST1000(iogrp),  rnacc*1e3/dtbgc,0.,cmpflg,'dustflx1000')
+    call wrtsrf(jdustflx2000(iogrp),  FLX_DUST2000(iogrp),  rnacc*1e3/dtbgc,0.,cmpflg,'dustflx2000')
+    call wrtsrf(jdustflx4000(iogrp),  FLX_DUST4000(iogrp),  rnacc*1e3/dtbgc,0.,cmpflg,'dustflx4000')
+    call wrtsrf(jdustflx_bot(iogrp),  FLX_DUST_BOT(iogrp),  rnacc*1e3/dtbgc,0.,cmpflg,'dustflx_bot')
```

Comment:
Did you check that the 1e3 is correct here? Usually this converts from kmol -> mol, is it here from kg -> g?

Reply:
Yes, it's from kg -> g.